### PR TITLE
VirusTotal Private API - add outputs and context to commands

### PR DESCRIPTION
**Pull request by bakatzir**
## Status
Ready

## Description
vt-private-get-url-report - add dropped files to outputs and context
vt-private-get-file-report - add ssdeep,authentihash,File type,imphash,File size,Magic literal,Tags.First seen,Community comment, community reputation to outputs and context.
vt-private-get-domain-report - add categories

Also, checking that these values are in the response to avoid getting errors.

## Related Issues
fixes: https://github.com/demisto/etc/issues/15706, https://github.com/demisto/etc/issues/15754

## Must have
- [x] Tests
- [x] Documentation (with link to it)
- [ ] Code Review

### CODE REVIEW

File: Integrations/integration-VirusTotal-Private_API.yml [hunk 1]
```diff
@@ -1358,19 +1421,22 @@ script:
...
     - contextPath: URL.VirusTotal.Resolutions.last_resolved
       description: Resolve date of IPs who resolved to this URL
-      type: Unknown
+      type: string
     - contextPath: URL.VirusTotal.Scans.Source
       description: Scan vendor for this URL
-      type: Unknown
+      type: string
     - contextPath: URL.VirusTotal.Scans.Detected
       description: Scan detection for this URL (True/False)
```
Comment (yuvalbenshalom) [inline Integrations/integration-VirusTotal-Private_API.yml]: this looks like a boolean value

File: Integrations/integration-VirusTotal-Private_API.yml [hunk 2]
```diff
@@ -1285,20 +1318,50 @@ script:
       type: number
     - contextPath: File.VirusTotal.Scans.Source
       description: Scan vendor for this hash
-      type: Unknown
+      type: string
     - contextPath: File.VirusTotal.Scans.Detected
       description: Scan detection for this hash (True,False)
```
Comment (yuvalbenshalom) [inline Integrations/integration-VirusTotal-Private_API.yml]: looks like a boolean value, need to check our policy on that

**Comment by bakatzir** [comment]
@yuvalbenshalom it's boolean.
fixed that and the outputs of all the other commands.

**Comment by bakatzir** [comment]
@yuvalbenshalom build fails on 1 of server versions, vt api quota issues.

**Pull request merged**